aws/lambda/pytorch-auto-revert/pytorch_auto_revert/__main__.py

```diff
@@ -159,6 +159,14 @@ def get_opts() -> argparse.Namespace:
             "If set, write the run state to HUD HTML; omit a value to use the run timestamp as the filename."
         ),
     )
+    workflow_parser.add_argument(
+        "--bisection-limit",
+        type=int,
+        default=None,
```

Comment:
Default should be what comes from the environment variable. We should maintain the standard here.